lib/venue.rb

```diff
@@ -1,2 +1,31 @@
 class Venue
+  attr_reader :name,
+              :capacity,
+              :patrons
+  def initialize(name, capacity)
+    @name     = name
+    @capacity = capacity
+    @patrons  = []
+  end
+
+  def add_patron(patron)
+    patrons << patron
+  end
+
+  def yell_at_patrons
+    yelled_at = []
+    @patrons.each do |patron|
+    yelled_at << patron.upcase
+    end
+    yelled_at
+  end
+
+  def over_capacity?
+    patrons.count > 4
```

Comment:
You are on the right track here. You are hard coding capacity by using `4`. Is this dynamic if you have less than `4`? You could check that it is `over_capacity` like so:
```
@patrons.length > @capacity
```
This will make it dynamic.